### PR TITLE
Project documentation (costin.curaru)

### PR DESCRIPTION
**Pull request by costinciuraru**
### Pull Request Overview

This pull request adds/changes/fixes...

### Testing Strategy

This pull request was tested by...

### TODO or Help Wanted

This pull request still needs...

### Build

- [ ] Ran `npm build`.

### Author
Costin Ciuraru

Signed-off-by: Costin Ciuraru <costin_ciuraru@yahoo.ro>

### CODE REVIEW

File: docs/project/2024/costin.ciuraru/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Drawbridge/Ramp
+A mechanism that can be used in a bigger scale as a drawbridge or in a smaller scale as a ramp.
+
+:::info 
+
+**Author**: Ciuraru Costin Cristian
```
Comment (alexandruradovici) [inline docs/project/2024/costin.ciuraru/index.md:6]:
```suggestion
**Author**: Ciuraru Costin Cristian \
```

File: docs/project/2024/costin.ciuraru/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+## Motivation
+
+I chose this project for seeing a lack of this type of mechanism in my country, drawbridges could be used in the future as we have a lot of rivers in our country. The project 
+could also be used as a ramp for people in wheelchairs to be able to enter buildings without hadicapped acces, for cars to be able to get over obstacles, and the posibilites are limitless.
+
+Being able to use my passion for electronics to help people and make life easier in general is amazing. This is a great oportunity to deepen my knowledge of 
+embedded systems and multiple mechanical and electrical parts, which are crucial in my studies and future career in technolo
+
+## Architecture 
+![projection](https://github.com/UPB-FILS-MA/upb-fils-ma.github.io/assets/163408586/84c0a003-3547-40d9-b80c-3d15bac355f2)
```
Comment (alexandruradovici) [inline docs/project/2024/costin.ciuraru/index.md:27]: Use the local file.

File: docs/project/2024/costin.ciuraru/index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+| Stepper Motor | Stepper Motor | [48,99 RON](https://www.optimusdigital.ro/ro/motoare-motoare-pas-cu-pas/3912-motor-pas-cu-pas-42hb34f08ab.html?search_query=stepper+motor&results=48) |
+
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (alexandruradovici) [inline docs/project/2024/costin.ciuraru/index.md:84]: Do you need these libraries?

File: docs/project/2024/costin.ciuraru/index.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [link](https://www.technoarete.org/common_abstract/pdf/IJERECE/v8/i8/Ext_18573.pdf)
+2. [link](https://www.youtube.com/watch?v=nydRO0k2aKY)
```
Comment (alexandruradovici) [inline docs/project/2024/costin.ciuraru/index.md:91]: Please add title to the links.

File: docs/project/2024/costin.ciuraru/index.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+| [LCD Display](https://www.waveshare.com/datasheet/LCD_en_PDF/LCD1602.pdf) | LCD Display| [30 RON](https://www.optimusdigital.ro/ro/optoelectronice-lcd-uri/62-lcd-1602-cu-interfata-i2c-si-backlight-galben-verde.html?search_query=lcd+1602&results=17) |
+| Stepper Motor | Stepper Motor | [48,99 RON](https://www.optimusdigital.ro/ro/motoare-motoare-pas-cu-pas/3912-motor-pas-cu-pas-42hb34f08ab.html?search_query=stepper+motor&results=48) |
```
Comment (alexandruradovici) [inline docs/project/2024/costin.ciuraru/index.md:75]: Please add a link to device.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/costin.ciuraru/index.md (hunk 5 above)
Comment (costinciuraru) [reply docs/project/2024/costin.ciuraru/index.md]: I added the link.

File: docs/project/2024/costin.ciuraru/index.md (hunk 4 above)
Comment (costinciuraru) [reply docs/project/2024/costin.ciuraru/index.md]: I added the title.

File: docs/project/2024/costin.ciuraru/index.md (hunk 3 above)
Comment (costinciuraru) [reply docs/project/2024/costin.ciuraru/index.md]: I changed the libraries.

File: docs/project/2024/costin.ciuraru/index.md (hunk 2 above)
Comment (costinciuraru) [reply docs/project/2024/costin.ciuraru/index.md]: I used the local file.

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-nita — APPROVED** [review]

**Pull request merged**